future-apps/payroll/contracts/Payroll.sol

```diff
@@ -649,8 +650,8 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
      * @param _employeeId Employee's identifier
      * @return Total amounts of previous and current owed salaries for the requested employee since their last payroll
      */
-    function _getOwedSalary(uint256 _employeeId) internal view returns (uint256 currentOwedSalary, uint256 totalOwedSalary) {
-        currentOwedSalary = _getCurrentOwedSalary(_employeeId);
+    function _getOwedSalaries(uint256 _employeeId) internal view returns (uint256 currentOwedSalary, uint256 totalOwedSalary) {
+        currentOwedSalary = _getCurrentCappedOwedSalary(_employeeId);
         totalOwedSalary = currentOwedSalary + employees[_employeeId].accruedSalary;
```

Comment:
Hmm, but wouldn't this be possible of overflowing?

Reply:
Yes, and we are performing a check to cap it again if that happens in the next line (see [L657](https://github.com/aragon/aragon-apps/pull/798/files#diff-9d7adaffbcce19867f027a983a30bcc7R657))

Reply:
🙈 missed that in the diff!